packaging/supervisor.conf

```diff
@@ -48,6 +48,15 @@ startretries=3
 priority=998
 user=dd-agent
 
+[program:dogstatsd6]
+command=/opt/datadog-agent/bin/dogstatsd6 start -l -f /etc/dd-agent/datadog.conf
+stdout_logfile=NONE
+stderr_logfile=NONE
+startsecs=2
+startretries=3
+priority=998
+user=dd-agent
```

Comment:
I _think_ (haven't actually checked what the binary does) that we also need to set `exitcodes=0` like the trace-agent does. By default supervisor also regards `2` as a successful exit code, and I think the go runtime can exit with `2` when an actual error occurs.